terraform/main.mk

```diff
@@ -36,29 +39,32 @@ terraform.init: gomplate terraform
 # TODO:? Implement -target approach so we can deploy specific apps only
 # TODO: generate env vars into tfvars in only one task
 terraform.apply: terraform.init ## Deploy infrastructure
-	@ cd $(INFRA_DIR)/env/$(ENV) && \
+	@ cd $(ENV_DIR) && \
 	$(TERRAFORM) plan -out=tfplan -input=false && \
 	$(TERRAFORM) apply -input=false tfplan && \
 	$(TERRAFORM) output -json > output.json
 
-terraform.test: terraform.init ## Test infrastructure
+terraform.checkov: ## Test infrastructure with checkov
 	$(CHECKOV)
```

Comment:
What does this do? Why do we need two separate commands?

Reply:
Fixed. Removed duplicate command.